css/theme.css

```
@@ -0,0 +1,1590 @@
+  align-items: center;
+  justify-content: center;
+}
+
+.theme-toggle {
+  cursor: pointer;
+  border-style: none;
+  background-image: none;
+  padding: 0px;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  line-height: 1;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

line-height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)